data/human/human missions.txt

```diff
@@ -6509,9 +6509,9 @@ event "kris gets frustrated"
 
 mission "Moving House 2"
 	minor
-	description "Kris is unsatisfied on <origin> and wants your help to move to <destination> instead."
+	description "Kris is unsatisfied with life on on <destination> and wants your help to move."
 	landing
-	name "Moving House, Again"
+	name "Moving House, again"
```

Comment:
If this is a title format, it should all be capitalized. If not, "House" should be "house."